### PR TITLE
Update OBJLoader2.html

### PR DESCRIPTION
**Pull request by bakerman**
Typo

### CODE REVIEW

File: docs/examples/loaders/OBJLoader2.html [hunk 1]
```diff
@@ -52,7 +52,7 @@ <h3>[name]( [param:LoadingManager manager], [param:LoaderSupport.ConsoleLogger l
 
 		<h2>Methods</h2>
 
-		<h3>[method:Object3D parse]( {[param:arraybuffer content]|[param:String content]] )</h3>
+		<h3>[method:Object3D parse]( [param:arraybuffer content]|[param:String content]] )</h3>
```
Comment (Mugen87) [inline docs/examples/loaders/OBJLoader2.html]: The last `]` in this line can also be removed in this case

Comment (michaelsauget) [inline docs/examples/loaders/OBJLoader2.html]: it should be

```suggestion
		<h3>[method:Object3D parse]( [param:arraybuffer content]|[param:String content] )</h3>
```

**Review by michaelsauget — CHANGES_REQUESTED** [review]

File: docs/examples/loaders/OBJLoader2.html (hunk 1 above)
Comment (Mugen87) [reply docs/examples/loaders/OBJLoader2.html]: @bakerman Can you please commit the suggestion by @michaelsauget?

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**